cve_bin_tool/cli.py

```diff
@@ -30,6 +30,7 @@
 import time
 from collections import ChainMap
 from pathlib import Path
+from platform import python_version
```

Comment:
Can we use sys.version_info instead?  We're already using it below for the importlib_metadata import.

Reply:
Sure, I'll do that